source/_integrations/nest.markdown

```diff
@@ -19,16 +19,18 @@ ha_domain: nest
 
 The `nest` integration allows you to integrate your [Google Nest](https://store.google.com/us/category/connected_home?) devices in Home Assistant. This integration uses the [Smart Device Management](https://developers.google.com/nest/device-access/api) API and Google's Cloud Pubsub to efficiently listen for changes in device state or other events.
 
-There is currently support for the following device types within Home Assistant:
+There is currently support for the following features within Home Assistant:
 
 - [Camera](#camera)
 - [Climate](#climate)
 - [Sensor](#sensor)
+- [Device Triggers](#automation-and-device-triggers)
```

Comment:
This list should contain device types supported. A TOC is provided on the right of each page, we don't need to create one.

Reply:
One thing I'm confused about is that the text says "device types" but then it lists some entities, which seemed inconsistent.  I was playing around with combining this with the list of supported devices in the overview, but it got a bit messy.  I'll just revert this all for now.

Reply:
Well, since we now have a TOC, we actually should start removing these all IMHO.
Instead, I can extract this data from the core source and add it to the frontmatter and show it automatically.

Future improvements 😉